### ISSUE TITLE
check the generation of the validation dataset

### ISSUE THREAD

**Issue by sonjageorgievska** [DeepRank/deeprank#22]
From Li: is the validation dataset made of models that do not appear in the training set?

**Comment by NicoRenaud** [comment]
That should be the case. Worth double checking

**Comment by LilySnow** [comment]
I personally think an ideal validation set probably should be generated on the case-level not model-level. For example, all the models from one specific case either belong to the training or the validation set, not both.

**Comment by LilySnow** [comment]
Done. Now the users can specify training, validation and test specifically.
